test/gracefulstop_test.go

```diff
@@ -121,14 +129,14 @@ func (s) TestGracefulStop(t *testing.T) {
 			return stream.Send(&testpb.StreamingOutputCallResponse{})
 		},
 	}
-	s := grpc.NewServer()
-	testgrpc.RegisterTestServiceServer(s, ss)
+	ss.S = grpc.NewServer()
+	stubserver.StartTestService(t, ss)
 
 	// 1. Start Server
 	wg := sync.WaitGroup{}
 	wg.Add(1)
 	go func() {
-		s.Serve(dlis)
+		ss.S.Serve(dlis)
```

Comment:
we should remove this as `stubserver.StartTestService` already starts the server